### PR TITLE
Kanban ticket view improvements

### PR DESCRIPTION
**Pull request by nachoserra-domatix**
Added improvements to the kanban ticket view like colors, progress bar, tags, state selection and user image.

_**Before**_
before

_**After**_
after

### CODE REVIEW

File: helpdesk_mgmt/README.rst [hunk 1]
```diff
@@ -172,6 +172,7 @@ Contributors
   * Carlos Martínez
   * Catalin Airimitoaie
   * Álvaro López
+  * Nacho Serra
```
Comment (pedrobaeza) [inline helpdesk_mgmt/README.rst]: Don't do this here, but on readme/CONTRIBUTORS.rst file

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please put both screenshots of before and after

**Comment by pedrobaeza** [comment]
Please don't do merge operations in the PR. Unscramble the commit history.

**Comment by nachoserra-domatix** [comment]
> Please put both screenshots of before and after

Done.

File: helpdesk_mgmt/README.rst (hunk 1 above)
Comment (nachoserra-domatix) [reply helpdesk_mgmt/README.rst]: Changed, sorry

File: helpdesk_mgmt/README.rst [hunk 2]
```diff
@@ -23,7 +23,7 @@ Helpdesk Management
     :target: https://runbot.odoo-community.org/runbot/282/11.0
     :alt: Try me on Runbot
 
-|badge1| |badge2| |badge3| |badge4| |badge5| 
+|badge1| |badge2| |badge3| |badge4| |badge5|
```
Comment (pedrobaeza) [inline helpdesk_mgmt/README.rst]: You still keep this unwanted change

File: helpdesk_mgmt/views/helpdesk_ticket_view.xml [hunk 3]
```diff
@@ -80,7 +80,7 @@
 
                             <field name="priority" widget="priority"/>
                             <field name="company_id" options="{'no_create': True}" groups="base.group_multi_company"/>
-                            <field name="create_date"/>
+                            <field name="create_date" readonly="1"/>
```
Comment (pedrobaeza) [inline helpdesk_mgmt/views/helpdesk_ticket_view.xml]: Wasn't this already changed previously?

File: helpdesk_mgmt/views/helpdesk_ticket_view.xml [hunk 4]
```diff
@@ -174,13 +174,49 @@
...
+                            <div class="o_kanban_record_bottom">
+                              <div class="oe_kanban_bottom_left"/>
+                              <div class="oe_kanban_bottom_right">
+                                <field name="activity_ids" widget="kanban_activity" related="activity_state"/>
+                                <field name="kanban_state" widget="state_selection"/>
+                                <field name="activity_state" invisible="1"/>
+                                <img t-att-src="kanban_image('res.users', 'image_small', record.user_id.raw_value)" t-att-title="record.user_id.value" width="24" height="24" class="oe_kanban_avatar"/>
+                              </div>
+                            </div>
+                            <!-- <div>
```
Comment (pedrobaeza) [inline helpdesk_mgmt/views/helpdesk_ticket_view.xml]: Always remove commented code

File: helpdesk_mgmt/views/helpdesk_ticket_view.xml (hunk 3 above)
Comment (nachoserra-domatix) [reply helpdesk_mgmt/views/helpdesk_ticket_view.xml]: My bad, solved.

**Review by pedrobaeza — APPROVED** [review]

**Review by samuel-domatix — APPROVED** [review]

**Comment by emagdalenaC2i** [comment]
@NachoSerra please see https://github.com/OCA/helpdesk/pull/87#issuecomment-530742866

**Comment by nachoserra-domatix** [comment]
> @NachoSerra please see [#87 (comment)](https://github.com/OCA/helpdesk/pull/87#issuecomment-530742866)

Done.

**Review by emagdalenaC2i — APPROVED** [review]

**Pull request merged**